support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -390,6 +390,12 @@ function CheckoutComponent({ geoId }: Props) {
 	const elements = useElements();
 	const cardElement = elements?.getElement(CardNumberElement);
 	const [stripeClientSecret, setStripeClientSecret] = useState<string>();
+	/**
+	 * flag to ensure the (successful) recaptcha token has been fully passed to Stripe
+	 * used to disable/set 'Loading...' message on Pay Now button (Playwright test use)
```

Comment:
```suggestion
	 * flag that disables the submission of the form until the stripeClientSecret is 
	 * fetched from the Stripe API with the reCaptcha token
```

Reply:
Ok, applied, thats a more technical description of the use of state `stripeClientSecretInProgress`